### PR TITLE
Add port 9081 to list of required ports

### PR DESCRIPTION
**Pull request by GitaShmueli**
Adding ports that are used for RS

### CODE REVIEW

File: content/rs/administering/designing-production/networking/port-configurations.md [hunk 1]
```diff
@@ -29,6 +29,8 @@ update your firewall with the port for that new database endpoint.
 | 8001 | Used by your application to access the RS [Discovery Service]({{< relref "/rs/concepts/data-access/discovery-service.md" >}}) | TCP |
 | 8443 | Used for secure (https) access to the management web UI | TCP |
 | 8444, 9080 | Used for nginx \<-\>cnm_http/cm communications on the same host only. Ports are bound to loopback adapter. | TCP |
+| 9081 | Used for crdb_coordinator | TCP |
```
Comment (Paz-Yanover) [inline content/rs/administering/designing-production/networking/port-configurations.md]: "Used for CRDB management"

Comment (Paz-Yanover) [inline content/rs/administering/designing-production/networking/port-configurations.md]: @ushachar - can you please approve these ports are new and required for RS deployments- 9081 (- CRDB management) and 8070, 8071 (- metrics exporter management)

Comment (Paz-Yanover) [reply content/rs/administering/designing-production/networking/port-configurations.md]: @RoeyPrat - can you approve it please? Maybe we can use a better explanation that does not use the internal "unknown" phrase of crdb coordinator?

Comment (Paz-Yanover) [reply content/rs/administering/designing-production/networking/port-configurations.md]: @RoeyPrat  ^^^^ (another two ports)

Comment (RoeyPrat) [reply content/rs/administering/designing-production/networking/port-configurations.md]: yes, 9081 is used by crdb_coordinator. @yossigo @guybe7 @oranagra  FYI

yes 8070, 8071 are used by metrics_exporter. @danni-m FYI

File: content/rs/administering/designing-production/networking/port-configurations.md [hunk 2]
```diff
@@ -29,6 +29,8 @@ update your firewall with the port for that new database endpoint.
 | 8001 | Used by your application to access the RS [Discovery Service]({{< relref "/rs/concepts/data-access/discovery-service.md" >}}) | TCP |
 | 8443 | Used for secure (https) access to the management web UI | TCP |
```
Comment (RoeyPrat) [inline content/rs/administering/designing-production/networking/port-configurations.md]: what does each row represent? 
I think each row could represent a system.
if this case:

- 8443 and 8444 should be in a row (CM)
- 9080 should be with cnm_api row (8080, 9443) and it's name should be cluster API

File: content/rs/administering/designing-production/networking/port-configurations.md (hunk 1 above)
Comment (yossigo) [reply content/rs/administering/designing-production/networking/port-configurations.md]: Note that it's a loopback port, used to communicate with nginx only and not exposed outside the node.

Comment (RoeyPrat) [reply content/rs/administering/designing-production/networking/port-configurations.md]: I think port that are used should be specified, regardless of whether they are exposed. 
Perhaps it is a good idea to specify which are exported in the table, or put the internal ones in a separate table.

File: content/rs/administering/designing-production/networking/port-configurations.md (hunk 2 above)
Comment (bmansheim) [reply content/rs/administering/designing-production/networking/port-configurations.md]: @RoeyPrat Agreed. Can you make that commit?

File: content/rs/administering/designing-production/networking/port-configurations.md (hunk 1 above)
Comment (bmansheim) [reply content/rs/administering/designing-production/networking/port-configurations.md]: @yossigo Which port is loopback?

File: content/rs/administering/designing-production/networking/port-configurations.md (hunk 2 above)
Comment (RoeyPrat) [reply content/rs/administering/designing-production/networking/port-configurations.md]: sure, but in a ticket - created RED-26867.
I suggest merging this for now, as it is an improvement on the existing.

File: content/rs/administering/designing-production/networking/port-configurations.md (hunk 1 above)
Comment (bmansheim) [reply content/rs/administering/designing-production/networking/port-configurations.md]: @RoeyPrat If loopback traffic doesn't pass a firewall between nodes then why list it?

Comment (yossigo) [reply content/rs/administering/designing-production/networking/port-configurations.md]: @bmansheim port 9081, used by crdb_coordinator.  Loopback local ports don't need to be punched through firewalls, but they may conflict other processes running on the same node so I see value in documenting them.

Comment (RoeyPrat) [reply content/rs/administering/designing-production/networking/port-configurations.md]: I agree with yossi on this.
install.sh checks if these ports are available, and fails installation if they are not.
It also points user to docs for explanation.

File: content/rs/administering/designing-production/networking/port-configurations.md [hunk 3]
```diff
@@ -5,49 +5,49 @@ weight: $weight
...
-| 20000-29999 | Used for internal communications with database shards | TCP |
+| ICMP | * | For connectivity checking between nodes |
+| TCP | 3333, 3334, 3335, 3336, 3337, 3338, 3339, 36379, 36380 | Internal cluster usage |
+| UCP | 53 | For accessing DNS/mDNS functionality in the cluster |
+| UCP | 5353 | For accessing DNS/mDNS functionality in the cluster |
+| TCP | 8001 | For application to access the RS [Discovery Service]({{< relref "/rs/concepts/data-access/discovery-service.md" >}}) |
+| TCP | 8443 | For secure (https) access to the management web UI |
+| TCP | 8444, 9080 | For nginx \<-\>cnm_http/cm communications on the same host only. Ports are bound to loopback adapter. |
+| TCP | 9081 | For CRDB management |
+| TCP | 8070, 8071 | For metrics exported and managed by nginx |
```
Comment (Paz-Yanover) [inline content/rs/administering/designing-production/networking/port-configurations.md]: I didn't understand the explanation of 8070 and 8071

File: content/rs/administering/designing-production/networking/port-configurations.md [hunk 4]
```diff
@@ -5,49 +5,49 @@ weight: $weight
...
-| 10000-19999 | Used for exposing databases externally | TCP |
-| 20000-29999 | Used for internal communications with database shards | TCP |
+| ICMP | * | For connectivity checking between nodes |
+| TCP | 3333, 3334, 3335, 3336, 3337, 3338, 3339, 36379, 36380 | Internal cluster usage |
+| UCP | 53 | For accessing DNS/mDNS functionality in the cluster |
+| UCP | 5353 | For accessing DNS/mDNS functionality in the cluster |
+| TCP | 8001 | For application to access the RS [Discovery Service]({{< relref "/rs/concepts/data-access/discovery-service.md" >}}) |
+| TCP | 8443 | For secure (https) access to the management web UI |
+| TCP | 8444, 9080 | For nginx \<-\>cnm_http/cm communications on the same host only. Ports are bound to loopback adapter. |
+| TCP | 9081 | For CRDB management |
```
Comment (Paz-Yanover) [inline content/rs/administering/designing-production/networking/port-configurations.md]: we should say something about the fact that this port is internal to the node and not required / used outside of the node

**Pull request merged**